gbdk-lib/examples/gb/banks_autobank/Makefile

```diff
@@ -1,30 +1,22 @@
-LCC	= ../../../bin/lcc -Wa-l -Wl-m
-BANKPACK  = ../../../bin/bankpack
+LCC	= lcc -Wa-l -Wl-m
 
-# uncomment for lcc verbose output
-# CFLAGS	= -v
+BINS = autobanks.gb
+CSOURCES = $(wildcard src/*.c)
+OBJS = $(addprefix obj/, $(notdir $(CSOURCES:.c=.o)))
 
-BINS	    = autobanks.gb
-CSOURCES   := $(wildcard *.c)
-ASMSOURCES := $(wildcard *.s)
-
-# Compiling will produce .o files
-OBJS       = $(CSOURCES:%.c=%.o) $(ASMSOURCES:%.c=%.o)
-
-all:	$(BINS)
+all: obj $(BINS)
```

Comment:
It would be good to keep .asm sources in the list so that the makefile would still work if they wanted to base a project on it that included asm source files.

Reply:
I think given the inconsistency of the makefiles it might be an idea to just provide a boilerplate one rather than making them *all* fully featured